docs/install.rst

```diff
@@ -0,0 +1,128 @@
+Installation
+============
+
+Install with pip
+----------------
+
+Note that to use ``astrowidgets`` with Jupyter Lab you need to
+install `nodejs from here <https://nodejs.org/en/download/>`_::
+
+    pip install astrowidgets
+    #
+    # ↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓↓
+    # INSTALL NODEJS FROM https://nodejs.org/en/download/
+    # ↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑↑
+    #
+    # (the nodejs on PyPI is *NOT* the right thing)
+    #
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+
+Install with conda
+------------------
+
+conda installation::
+
+    conda install -c conda-forge astrowidgets nodejs
+    jupyter labextension install @jupyter-widgets/jupyterlab-manager
+    # Optional, but avoids a rebuild when you open lab
+    jupyter labextension build
+
+.. note::
+
+    If you are using a virtual environment and
+    are experiencing problems with the installation, be sure to check and use
+    the suitable ``pip`` version. Also, if you are having problem even with the following
+    dependencies installed, please make sure you have the most up-to-date versions.
+    Upgrade dependencies when and where necessary.
+
+The following dependencies are needed *in the kernel of execution*
+to use ``astrowidgets`` in either Jupyter Lab or Notebook. They should be installed
+automatically when you install astrowidgets:
+
+* ``python>=3.6``
+* ``numpy``
+* ``astropy``
+* ``ipywidgets>=7.5``
+* ``ipyevents>=0.6.3``
+* ``ginga>=2.7.1``
+* ``pillow``
+* ``freetype``
+* ``aggdraw``
+* ``jupyterlab>=1``
+* ``opencv`` (optional, not installed by default)
+* ``nodejs`` (only if you plan to use Jupyter Lab, not installed by default)
```

Comment:
So, I still thinks the instruction is a bit confusing because `jupyterlab` is installed regardless as a requirement but here the text suggests that it is an optional dependency.

Reply:
Fixed, I think...